install/deploy_management_nodes.md

```diff
@@ -106,7 +106,14 @@ proceed to step 2.
 
    This ensures that the PIT is configured with an accurate date/time, which will be properly propagated to the NCNs during boot.
 
-1. Ensure the current time is set in BIOS for all management NCNs.
+   If you receive the error `Failed to set time: NTP unit is active` you will need to stop `chrony` first.
+
+   ```bash
+   pit# systemctl stop chronyd
+   ```
+   Then run the commands above to complete the process.
+
+3. Ensure the current time is set in BIOS for all management NCNs.
```

Comment:
I think this needs to be a `1` not a `3`.  The various Markdown renderers will do the right thing.